From b0c367e43f67bbe0c23d60abd4eff1766597f0b8 Mon Sep 17 00:00:00 2001
From: Binh Nguyen <binh.nguyen.uw@renesas.com>
Date: Fri, 29 Mar 2019 10:53:02 +0700
Subject: [PATCH 159/175] clk: r8a774c0-cpg-mssr: add vcp4, fdp clock

this commit add:
- VCP4(VCPLF), VCP4(VDPB) clocks
- fdp clock

Signed-off-by: Binh Nguyen <binh.nguyen.uw@renesas.com>
---
 drivers/clk/renesas/r8a774c0-cpg-mssr.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/clk/renesas/r8a774c0-cpg-mssr.c b/drivers/clk/renesas/r8a774c0-cpg-mssr.c
index b0667e6..f24fe4a 100644
--- a/drivers/clk/renesas/r8a774c0-cpg-mssr.c
+++ b/drivers/clk/renesas/r8a774c0-cpg-mssr.c
@@ -124,6 +124,9 @@ static const struct cpg_core_clk r8a774c0_core_clks[] __initconst = {
 
 static const struct mssr_mod_clk r8a774c0_mod_clks[] __initconst = {
 	DEF_MOD("3dge",                  112,   R8A774C0_CLK_ZG),
+	DEF_MOD("fdp1-0",                119,   R8A774C0_CLK_S0D1),
+	DEF_MOD("vcplf",                 130,   R8A774C0_CLK_S3D1),
+	DEF_MOD("vdpb",                  131,   R8A774C0_CLK_S3D1),
 	DEF_MOD("tmu4",                  121,   R8A774C0_CLK_S0D6C),
 	DEF_MOD("tmu3",                  122,   R8A774C0_CLK_S3D2C),
 	DEF_MOD("tmu2",                  123,   R8A774C0_CLK_S3D2C),
-- 
2.7.4

